classes/controller/AdminController.php

```diff
@@ -1666,7 +1666,7 @@ public function initPageHeaderToolbar()
             $this->page_header_toolbar_title = $this->toolbar_title[count($this->toolbar_title) - 1];
         }
 
-        $this->context->smarty->assign('help_link', 'https://help.prestashop.com/' . Language::getIsoById($this->context->employee->id_lang) . '/doc/'
+        $this->context->smarty->assign('help_link', 'https://help.prestashop-project.org/' . Language::getIsoById($this->context->employee->id_lang) . '/doc/'
```

Comment:
@PrestaShop/product-team Is it possible to have a configuration and not harcoded value ?

Reply:
I think the issue is that this is legacy code, we still don't have a global common way to configure properly Symfony and legacy code

Reply:
I already tried that and all tests are failing. Therefore, as it is needed for the beta, I just updated the link. There should be a way for calling Symfony services directly from legacy controllers.

If I can, i'll work on it, but will be in another PR

Reply:
For 8.0.0, it will be perfect for SMB edition.